buttercup/strings/en_US.yaml

```diff
@@ -170,6 +170,8 @@ rules:
     Sorry, but it looks like r/{0} has gone private!
   no_rules: |
     r/{0} doesn't appear to have any rules!
+  no_filter_rules: |
```

Comment:
If the message is the same, do we need to have it duplicated or can we just return the no rules message?

Reply:
I duplicated it because it is called via a localization key us function parameter. It's a bit hacky, but it allows us to use the same function for the `/rules` and `/pirules` command.

Not sure if there is a better way to handle this

Reply:
Ah. If it's the best way to handle it for now, then we'll keep it 👍